src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/DenormalizedPredicateRewriter.cs

```diff
@@ -22,7 +22,7 @@ internal class DenormalizedPredicateRewriter : ExpressionRewriterWithInitialCont
 
         public Expression VisitSqlRoot(SqlRootExpression expression, object context)
         {
-            if (expression.TableExpressions.Count == 0 || expression.DenormalizedExpressions.Count == 0 || expression.TableExpressions.All(t => t.Kind == TableExpressionKind.Chain))
+            if (expression.TableExpressions.Count == 0 || expression.DenormalizedExpressions.Count == 0 || expression.TableExpressions.All(t => t.Kind == TableExpressionKind.Chain) || expression.TableExpressions.All(t => t.Kind == TableExpressionKind.Include))
```

Comment:
`expression.TableExpressions.All(t => t.Kind == TableExpressionKind.Chain || t.Kind == TableExpressionKind.Include)`?

Reply:
This will break a query such as `Observation?patient:Patient.name=Adams&_include=Observation:subject` where everything is one or other.